### DISCUSSION TITLE
Discussion about Windows Restricted Traffic Limited Functionality Baseline for Windows privacy.

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#448] [General]
This [article](https://learn.microsoft.com/en-gb/windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services) by Microsoft talks about managing connections made by the Windows operating system. 

The article mentions a [Windows Restricted Traffic Limited Functionality Baseline](https://download.microsoft.com/download/D/9/0/D905766D-FEDA-43E5-86ED-8987CEBD8D89/WindowsRTLFB.zip). There are both security and usability concerns with applying this baseline but I would like to hear thoughts on it anyway. 

Would this project benefit from having a configurable GUI for this baseline so users can eliminate any connections they may deem unnecessary?

**Comment by ghost** [comment]
Relevant: 
https://learn.microsoft.com/en-gb/windows/privacy/manage-windows-11-endpoints

**Comment by HotCakeX** [comment]
Thanks for the suggestion, I'll implement it in Harden Windows Security module.
